ccpage40.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#3-1. Names: Store the names of a few of your friends in a list called names. 
+# Print each person’s name by accessing each element in the list, one at a time.
+names = ['fred', 'tom', 'sam']
+print(names[0].capitalize())
+print(names[1].capitalize())
+print(names[-1].capitalize())
+
+#3-2. Greetings: Start with the list you used in Exercise 3-1, but instead of just printing each person’s name, print a message to them.
+print(f"Welcome to Texas, {names[0].capitalize()}!")
+print(f"{names[1].capitalize()}, here's the list of ingredients:\nmilk,\nbread, and\ncheese.")
+print(f"{names[-1].capitalize()}, let's go to the game today.")
```

Comment:
I'd personally do `names[2]` here as it matches how you did lines 17-18, but that's nit-picking.